File: quick_sort/quickSort.py
def quickSort(nums):
    if len(nums) < 2:
        return nums

    pivot = nums[len(nums) - 1]
    left = []
    right = []

    for i in range(len(nums) - 1):
        if nums[i] < pivot:
            left.append(nums[i])
        else:
            right.append(nums[i])
    return quickSort(left) + [pivot] + quickSort(right)

File: quick_sort/quickSort.js
const quickSort = (nums) => {
  if (nums.length < 2) {
    return nums;
  }

  const pivot = nums[nums.length - 1];
  let left = [];
  let right = [];

  for (let i = 0; i < nums.length - 1; i++) {
    if (nums[i] < pivot) {
      left.push(nums[i]);
    } else {
      right.push(nums[i]);
    }
  }

  left = quickSort(left);
  right = quickSort(right);

  return [...left, pivot, ...right];
};
